quickstart/quickstart.sh

```diff
@@ -33,6 +33,7 @@ oc new-project $MODELMESH_PROJECT
 oc apply -f ../manifests/kfdef.yaml -n $MODELMESH_PROJECT
 
 oc new-project $INFERENCE_SERVICE_PROJECT
+oc label namespace $INFERENCE_SERVICE_PROJECT "modelmesh-enabled=true" --overwrite=true || echo "INFO: modelmesh-enabled label already exists."
```

Comment:
> echo "INFO: modelmesh-enabled label already exists." 

That is not the only reason an error could occur (e.g. permission issues), I would use a broader message like "failed to apply label"

Reply:
Fixed :-)

Reply:
Fixed :-)